open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<p>Using ":::" instead of "<code></code>`"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
DONE